cmd/mycelo/genesis_commands.go

```diff
@@ -185,8 +188,6 @@ func envFromTemplate(ctx *cli.Context, workdir string) (*env.Environment, *genes
 		}
 	}
 
-	genesisConfig.Hardforks.GingerbreadBlock = gingerbreadBlock
```

Comment:
If you remove this line you won't be setting the block.
The first if above is required because the `baseConfig` requires the gingerblock

Reply:
And that baseConfig gives you the genesisConfig which you are setting. I'm changing it also with the default in 1 for the P2

Reply:
That's also where the forks get set: https://github.com/celo-org/celo-blockchain/blob/925fba88d7f84bc5e35c776ee73258eb9f8df8aa/mycelo/genesis/genesis.go#L43-L44

There's some additional logic that sets them to 1, when no other value is given. So in case of not passing the argument, we would overwrite with `nil` here, oven though it was set before. That's the whole problem this PR fixes.

Reply:
ohhhh, missed that one